content/docs/introduction/billing.md

```diff
@@ -0,0 +1,188 @@
+|**Storage**               | 3 GB per branch                   | 200 GB        | Unlimited                 |
+|**Dedicated resources**   | -                            | -           | &check;                   |
+|**Project sharing**       | -                            | &check;          | &check;                   |
+|**Auto-suspend compute**  | &check;                           | &check;          | &check;                   |
+|**Configurable auto-suspend compute (coming soon)**  | -                           | &check;          | &check;                   |
+|**Always-on compute (coming soon)**     | -                            | &check;          | &check;                   |
+|**Autoscaling (coming soon)**     | -                            | &check;          | &check;                   |
+|**Payment**               | Free                              | Credit Card, Pay As You Go with monthly invoicing | Prepaid, Custom Contract, Volume Discounts |
+|**Support**               | Community, support tickets                 | Community, support tickets, video chat          | Community, support tickets, video chat, resale customer support                   |
+
+<Admonition type="info">
+The limits described above are plan defaults. If you would like to adjust the limits to tailor a plan to your specific requirements, please contact [sales@neon.tech](mailto:sales@neon.tech).
+</Admonition>
+
+Our [Pricing](https://neon.tech/pricing) page provides additional information and a calculator you can use to estimate costs and determine which plan is right for you.
+
+## Account billing page
+
+Each Neon account has a billing page, where you can:
+
+- View your current billing total for the month-to-date, including a cost breakdown by [billing metric](#neon-billing-metrics).
+- Update your payment details
+- Download your latest invoices
+
+To access your billing page:
+
+1. Navigate to the Neon Console.
+1. Select **Billing** from the sidebar.
+
+## Neon invoices
+
+A Neon invoice includes an **Amount due** for billing period and the cost broken down by [billing metric](#neon-billing-metrics).
+
+### Download invoices
+
+You can download invoices from the **Billing** page.
+
+1. Navigate to the **Billing** page in the Neon Console. 
+1. Under **Latest invoices**, locate the invoice you want to download and click the PDF download icon.
+
+## Cancel a subscription
+
+To cancel your subscription to a Neon paid plan:
+
+1. Navigate to the **Billing** page in the Neon Console.
+1. Click **Cancel subscription**.
+1. Enter your cancellation request and click **Submit**.
+
+This action initiates the cancellation. If your data exceeds [Free Tier](/docs/introduction/technical-preview-free-tier) storage limits, you will be contacted by the Neon Support team with a request to reduce your storage before the paid plan is canceled and Free Tier limits are applied.
```

Comment:
```suggestion
This action initiates the cancellation. If your data exceeds [Free Tier](/docs/introduction/technical-preview-free-tier) storage limits, you will need to reduce your storage before the paid plan is canceled and Free Tier limits are applied. Your request will be processed by the Neon Support team.
```

rewording slightly so that a reader does not think they need to wait for the support person to reach out to start reducing their usage